Update vendor-portal-creating-account.md (#633)
replicatedhq/replicated-docs: docs/vendor/vendor-portal-creating-account.md

  
    - Copy and paste the activation code into the text box and click **Activate**. Your account is now activated.

      :::note

>       Once your account is activated, you may have the option to join a team that you have an existing invite for or to [Auto-Join](/vendor/team-management#enable-users-to-auto-join-your-team) an existing team.

Reviewer: ```suggestion
      Once your account is activated, you may have the option to accept a pending invite, or to [Auto-Join](/vendor/team-management#enable-users-to-auto-join-your-team) an existing team.
```
